pkg/artifact_builder/docker_compose.go

```diff
@@ -47,7 +51,14 @@ func (bc *BuilderConfig) invokeDockerCompose(command DockerCommand) ([]byte, err
 
 	envVars := bc.GetBuildEnv()
 	envVars = append(envVars, os.Environ()...)
-	envVars = append(envVars, "DOCKER_BUILDKIT=1")
+
+	// Specifying platform in the docker compose conflicts with the DOCKER_DEFAULT_PLATFORM env var
```

Comment:
what does this conflict look like? an error? or is the env variable ignored in favor of that in the config?